spring-cloud-dataflow-platform-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/server/config/cloudfoundry/CloudFoundryPlatformProperties.java

```diff
@@ -39,6 +40,8 @@
 
 		private CloudFoundryDeploymentProperties deployment;
 
+		private CloudFoundrySchedulerProperties scheduler;
```

Comment:
Would be better to use `schedulerProperties` instead of just `scheduler` as `getScheduler()` returns just the properties ;)

Reply:
Hee I agree :-)